### PR TITLE
Build: update lock file

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:
Follow-up from #17707
#### Jetpack product discussion
* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Does it build?

#### Proposed changelog entry for your changes:

* N/A

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12287,12 +12287,12 @@ minimatch@3.0.4, minimatch@^3.0.0, minimatch@^3.0.2, minimatch@^3.0.3, minimatch
   dependencies:
     brace-expansion "^1.1.7"
 
-minimist@0.0.8, minimist@^0.2.1:
+minimist@0.0.8, minimist@^0.2.1, minimist@^1.2.5:
```
Comment (brbrr) [inline yarn.lock:12290]: That change is a bit weird. 1.2.5 should be resolved with 1.2.5 version, not the 0.2.1

**Review by brbrr — APPROVED** [review]
It builds, although I'm not sure this change makes a lot of sense. Although, I don't think it would affect our build process in any way.

File: yarn.lock (hunk 1 above)
Comment (jeherve) [reply yarn.lock:12290]: Maybe we don't need the minimist resolution here anymore? 
https://github.com/Automattic/jetpack/blob/159b0a850eef5ffd0de668cb12999adbcd71ceca/package.json#L105

If I remove it, here is the generated change:

```diff
diff --git a/yarn.lock b/yarn.lock
index 0e604748c..5f68c57b6 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -12287,12 +12287,12 @@ minimatch@3.0.4, minimatch@^3.0.0, minimatch@^3.0.2, minimatch@^3.0.3, minimatch
   dependencies:
     brace-expansion "^1.1.7"

-minimist@0.0.8, minimist@^0.2.1, minimist@^1.2.5:
-  version "0.2.1"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.2.1.tgz#827ba4e7593464e7c221e8c5bed930904ee2c455"
-  integrity sha512-GY8fANSrTMfBVfInqJAY41QkOM+upUTytK1jZ0c8+3HdHrJxBJ3rF5i9moClXTE8uUSnUo8cAsCoxDXvSY4DHg==
+minimist@0.0.8:
+  version "0.0.8"
+  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.8.tgz#857fcabfc3397d2625b8228262e86aa7a011b05d"
+  integrity sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=

-minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0:
+minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
   version "1.2.5"
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
   integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
```

**Comment by jeherve** [comment]
r216323-wpcom

**Pull request merged**